packages/components/src/actions.ts

```diff
@@ -1182,27 +1187,31 @@ export function getSelection(location: any): Promise<ISelectionResponse> {
     });
 }
 
-export function getSelectedData(model: QueryGridModel, columns?: List<QueryColumn>): Promise<IGridResponse> {
-    let filters = model.getFilters();
-    filters = filters.push(Filter.create('RowId', model.selectedIds.toArray(), Filter.Types.IN));
-
-    // If columns defined use those for the query columns else use the display columns
-    const columnString = columns ? columns.map(c => c.fieldKey).join(',') : model.getRequestColumnsString();
+export function getSelectedData(schemaName?: string,
```

Comment:
Should we add a method on QueryModel instead? It seems to me QueryModel already has the data as well as selection to be able to return selectedData, without having to call selectRows again.

Reply:
QueryModel has the data for the current page only.  It has all the selectedIds, but we need all the actual records.

I debated adding this as an action in QueryModel, but it seems a little outside of that since the full set of selected data is not something stored on the QueryModel itself and also not something that's normally needed.  It could be, of course, so I'm open to making that change.

Reply:
I see, fetching the full set of data to QueryModel probably have performance implication. What's currently implemented is probably a better trade off.